src/verify.c

```diff
@@ -474,6 +478,29 @@ static void add_missing_files(struct manifest *official_manifest)
 		if ((ret != 0) || hash_needs_work(file, local.hash)) {
 			file_not_replaced_count++;
 			printf("\n\tnot fixed\n");
+
+			if (cmdline_option_install && no_freespace_flag == false) {
+				fs_free = get_available_space(path_prefix);
```

Comment:
Can you please refactor lines 483-503 into its own function please? It's getting hard to read with the deep indentation at some parts.

Reply:
maybe 483-499, whatever makes sense.